README.md

```diff
@@ -1,24 +1,33 @@
-# Satellite Ansible Roles
-### Note: The roles here are not officially supported by Red Hat.
+# Satellite Ansible Collection(devel)
```

Comment:
I have a hard time calling this a collection (in the ansible context) while not uploading it to galaxy, but I think we agreed that this is a staging/devel place which we don't want to publish to Galaxy.

Mind you, it's just about the work collection as it has a very special meaning in Ansible land.

On the other hand, I think collection is what we're going to use when consuming this in Ansible, so maybe it's OK?

Reply:
I think whatever we go with we should rename the repository to match:

 * Collection
 * Assets
 * Junk drawer
 * Stuff
 * Compilation
 * Accumulation

Reply:
yeah I think we can call it collection, make changes, make it better and later add it to galaxy. It doesn't have to be right away.

Reply:
I'm happy if @ehelms or @evgeni can rename the repo. Thanks.

Reply:
I like compilation, but fine with either.

If we want to rename (which sounds like a good idea), it can't be collection, as satellite-ansible-collection already exists.

Reply:
I agree with @evgeni  about the name to be Compilation

Reply:
Three votes for compilation... let's roll with that as naming can always be a long conversation and until we are sure how we want to package this up (as a whole or split it up into logical collections) let's avoid 'collection' given that is a "branded" term in Ansible land.

Reply:
:+1:

Reply:
@kedark3 Can you update the wording here before merging from Collection to Compilation?

Reply:
@ehelms will do thanks :)

Reply:
Updated. There are instances in Readme that say it is a collection which I think is still correct.